Resources/Prototypes/_RMC14/Entities/Objects/Misc/hollow_cane.yml

```
@@ -0,0 +1,31 @@
+﻿- type: entity
+  parent: Cane
+  id: RMCHollowCane
+  name: hollow cane
```

```suggestion
  name: fancy cane
  description: An ebony cane with a fancy, seemingly-golden tip. Feels hollow to the touch.
  suffix: Hollow
```
parity name and description